drivers/mspi/mspi_dw.c

```diff
@@ -162,18 +173,18 @@ static void call_user_callback_with_context(const struct device *dev,
 					    uint32_t packet_idx,
 					    int status)
 {
+	LOG_DBG("Calling user function with evt_type: %u", evt_type);
 	struct mspi_dw_data *dev_data = dev->data;
 	const struct mspi_xfer_packet *packet =
 		&dev_data->xfer.packets[packet_idx];
 	struct mspi_callback_context *cb_ctx = dev_data->cb_ctxs[evt_type];
 
 	if (!(packet->cb_mask & BIT(evt_type)) ||
 	    !dev_data->cbs[evt_type]) {
+		LOG_DBG("Callback type not setup");
```

Comment:
Is this really valuable to always log "Calling user function ..." and additionally the above if there's no callback? If so, these changes should be moved to the commit that introduces `call_user_callback_with_context()`.